### DISCUSSION TITLE
Is there a way to get pos tagging label scheme list?

### DISCUSSION THREAD

**Discussion by Boltzmachine** [explosion/spaCy#7896] [Help: Coding & Implementations] [answered]
I did not find it in [https://spacy.io/models/en](https://spacy.io/models/en). And I do not find it in `nlp.pipe_labels`.

**Comment by adrianeboyd** [comment, marked as answer]
Hi, the POS tags are the universal dependencies "universal POS" tags: https://universaldependencies.org/docs/u/pos/

`spacy` uses the additional tag `SPACE` for whitespace tokens. The formal list within spacy is here:

https://github.com/explosion/spaCy/blob/df3444421aba611d4ad1238610ce189df158d85a/spacy/parts_of_speech.pyx

**Reply by Boltzmachine** [reply]
Thanks. So can I read it with spacy in my python codes?

**Reply by adrianeboyd** [reply]
Sure, it's very stable. The last time it changed was when UD v2.0 was introduced in 2017 and there were some minor modifications to the tag set.

**Reply by Boltzmachine** [reply]
Thanks!
